lightwood/api/json_ai.py

```diff
@@ -750,8 +759,14 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
     # Pre-processing Body
     # ----------------- #
 
+    imputers = ""
+    for target_col, imputer in json_ai.imputers.items():
+        imputers += f"{target_col}: {imputer},"
+    imputers = "{" + imputers + "}"
```

Comment:
We have an `inline_dict` function that would help here (I think) + I think this and the loop at line 669 can be one and the same, with the imputers being constructed with `call` here